primefaces-integration-tests-jakarta/src/test/resources-filtered/primefaces-selenium/config.properties

```diff
@@ -7,3 +7,5 @@ timeout.documentLoad = 30
 
 webdriver.browser = ${webdriver.browser}
 webdriver.headless = ${webdriver.headless}
+
+junit.path4Screenshots = ${JUNIT_PATH4SCREENSHOTS}
```

Comment:
What about "screenshotDirectory" instead of "junit...."?

Reply:
done - renamed at all necessary places